services/course-material/src/components/ContentRenderer/moocfi/TerminologyBlock.tsx

```
@@ -56,7 +59,11 @@ const TerminologyBlock: React.FC<
       >
         {props.data.attributes.blockName}
       </span>
-      <h2>{props.data.attributes.title}</h2>
+      <h2
+        dangerouslySetInnerHTML={{
```

Biome

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)